service_crategen/src/commands/generate/codegen/rest_xml.rs

```diff
@@ -163,6 +163,7 @@ impl GenerateProtocol for RestXmlGenerator {
 fn generate_documentation(operation: &Operation) -> String {
     match operation.documentation {
         Some(ref docs) => {
+            println!("\n\ndocs are {:?}\n\n", docs);
```

Comment:
Leftover line to extract the string we are doing replacing on.  Tested to see if using a regex was faster and for our small strings it was not.